server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -669,14 +665,13 @@ private HostAndPort startStatsService() {
   /**
    * Checks if the system is almost out of memory.
    *
-   * @param runtime
-   *          Java runtime
+   * @param candidateLength
+   *          Candidate Length
    * @return true if system is almost out of memory
-   * @see #CANDIDATE_MEMORY_PERCENTAGE
+   * @see #CANDIDATE_BATCH_SIZE
    */
-  static boolean almostOutOfMemory(Runtime runtime) {
-    return runtime.totalMemory() - runtime.freeMemory()
-        > CANDIDATE_MEMORY_PERCENTAGE * runtime.maxMemory();
+  static boolean almostOutOfMemory(Long candidateLength) {
```

Comment:
I don't think the method comment is true anymore.  Perhaps this method name and comments should be changed altogether.

Reply:
That's fair, I wasn't quite sure on that either. Do you have any suggestions?

Reply:
The method is simple now, I would inline it and remove the associated test.

Reply:
That is what I originally had but was unsure on if removing the test was a good idea.